installer/roles/mobile-control-panel-setup/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
@maleck13 There is an official repo for installing openshift via Ansible. See[ openshift-ansible](https://github.com/openshift/openshift-ansible).

This repository includes specific roles to install Ansible with features enabled. Take metrics and centralised logging for example. It also includes custom libraries for working with OpenShift objects.

The thinking is that if we were to work off a fork of both that and origin while developing mobile it would allow us to follow best practices from the off and give us a seamless integration at a later stage. It would also allow us to make use of the modules that are being actively developed and contribute back where we can.

Any thoughts, comments or concerns ?